colin/core/config/config.py

```diff
@@ -0,0 +1,23 @@
+from colin.core.loader import load_check_implementation
+from colin.core.target import is_compatible
+
+
+class Config(object):
+
+    def __init__(self, name=None):
+        self.name = name or "default"
+        # load from /usr/share/colin/${name}
+
+    def get_checks(self, group, target_type, severity=None, tags=None):
```

Comment:
What does it mean `severity`?

Reply:
required x optional x warn
("vážnost"...;-)

Any suggestions for the better name?
It would be nice to specify possible options.

Reply:
severity is good, that's a common name for such property